src/main/java/org/onedatashare/server/controller/MkdirController.java

```diff
@@ -46,6 +46,10 @@ public Object mkdir(@RequestHeader HttpHeaders headers, @RequestBody UserAction
       if (userAction.getCredential() == null) {
         return new ResponseEntity<>(new AuthenticationRequired("oauth"), HttpStatus.INTERNAL_SERVER_ERROR);
       } else return gridService.mkdir(cookie, userAction);
+    }else if("box:///".equals(userAction.type)) {
```

Comment:
This literal can be replaced with the ODSConstants reference

Reply:
Good catch, I missed that one